src/pages/index.js

```diff
@@ -1,38 +1,59 @@
 import React from "react"
+import apiClient from "panoptes-client/lib/api-client"
 import ReactGA from "react-ga"
```

Comment:
Seeing console warning `ReactGA.initialize must be called first or GoogleAnalytics should be loaded manually`, is it  being loaded manually?

Reply:
This isn't an issue because it only occurs on staging, where we don't care about logging, but the error disappears when using `gatsby build && gatsby serve`, which will log successfully to production.

However, I found in troubleshooting this that the app will not build without some changes I'll have to make.